lib/cocoapods/target/build_settings.rb

```diff
@@ -371,7 +378,16 @@ def add_inherited_to_plural(hash)
           if PLURAL_SETTINGS.include?(key)
             raise ArgumentError, "#{key} is a plural setting, cannot have #{value.inspect} as its value" unless value.is_a? Array
 
-            value = "$(inherited) #{quote_array(value)}"
+            if QUOTE_WRAPPED_SETTINGS.include?(key)
+              if value.empty?
+                value = "$(inherited)"
+              else
+                value_to_wrap = "$(inherited) #{quote_array(value)}".strip
```

Comment:
do the quotes from `quote_array` need to be escaped?

Reply:
I don't think so. It would make sense to need that, but this doesn't seem to work the same as the other settings anyway. From testing by manually changing the xcconfig to see what works...

Quotes around the individual path only doesn't build:
```
SWIFT_INCLUDE_PATHS = $(inherited) "$(PLATFORM_DIR)/Developer/usr/lib"
```

No quotes doesn't build:
```
SWIFT_INCLUDE_PATHS = $(inherited) $(PLATFORM_DIR)/Developer/usr/lib
```

Quotes around the whole paths array, but no quotes internally does build:
```
SWIFT_INCLUDE_PATHS = "$(inherited) $(PLATFORM_DIR)/Developer/usr/lib"
```

Adding quotes around the quoted paths does build:
```
SWIFT_INCLUDE_PATHS = "$(inherited) "$(PLATFORM_DIR)/Developer/usr/lib""
```

From looking at the diff in `cocoapods-integration-specs` after running `bundle exec rake spec:rebuild_integration_fixtures`, we're outputting the last one.

Reply:
that's... super super weird. thanks for checking!!